app/controllers/application_controller.rb

```
@@ -15,4 +15,12 @@ def configure_permitted_parameters
     devise_parameter_sanitizer.for(:sign_up) << :last_name
     devise_parameter_sanitizer.for(:sign_up) << :phone
   end
+
+  def authenticate_user!
+    begin
+      super
```

This is hiding the error.
Even if you do this because you don't find any other method you should document it throughly